gwvolman/fs_container.py

```diff
@@ -0,0 +1,75 @@
+# Description: Context manager for changing the current working directory
+
+import time
+import docker
+import requests
+
+from .constants import VOLUMES_ROOT
+from .utils import stop_container
+
+
+class FSContainer(object):
+    @staticmethod
+    def start_container(name):
+        cli = docker.from_env()
+        # Create container for handling FUSE mounts
+        print("Creating WT Filesystem container...")
+        fscontainer = cli.containers.run(
+            image="wholetale/girderfs",
+            name=name,
+            detach=True,
+            labels={"traefik.enable": "false"},
```

Comment:
Maybe add labels to relate the container to the running instance to simplify debugging -- or is there a way to relate them already?